NiL.JS/BaseLibrary/Array.cs

```diff
@@ -63,6 +63,15 @@ public Array(double length)
         }
 
         [DoNotEnumerate]
+        internal Array(JSValue[] data)
```

Comment:
Do you really need this constructor? Is `public Array(IEnumerable source)` not suitable?
Also this constructor cause an error if becomes public, because it does not clone values.
And if it is internal, then `[DoNotEnumerate]` is useless.